README.md

```diff
@@ -102,16 +111,28 @@ logging_enabled: true
 logging_purge_confs: true
 ```
 
-2) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and outputs into the local files.
+2) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and outputs into the local files, which actions are predefined.
```

Comment:
that sounds strange, what does it mean? @richm could you help with improving the wording, please?

Reply:
Maybe something like `... handle inputs from the local system (e.g. systemd journald), and outputs into local files (e.g. /var/log/messages).` and omit the `, which actions are predefined.` clause.

Reply:
Thank you, @richm.  Replacing the line.

Reply:
Done.